armi/nucDirectory/elements.py

```diff
@@ -90,41 +90,41 @@
      <Element  LR (Z=103), Lawrencium, ChemicalGroup.ACTINIDE, ChemicalPhase.SOLID>]
 
 
-For specific data on nuclides within each element, refer to the 
-:ref:`nuclide bases summary table <nuclide-bases-table>`.
-
-
-.. exec::
-    from tabulate import tabulate
-    from armi.nucDirectory import elements
-
-    attributes = ['z',
-                  'name',
-                  'symbol',
-                  'phase',
-                  'group',
-                  'is naturally occurring?',
-                  'is heavy metal?',
-                  'num. nuclides',]
-
-    def getAttributes(element):
-        return [
-            f'``{element.z}``',
-            f'``{element.name}``',
-            f'``{element.symbol}``',
-            f'``{element.phase}``',
-            f'``{element.group}``',
-            f'``{element.isNaturallyOccurring()}``',
-            f'``{element.isHeavyMetal()}``',
-            f'``{len(element.nuclides)}``',
-        ]
-
-    sortedElements = sorted(elements.byZ.values())
-    return create_table(tabulate(tabular_data=[getAttributes(elem) for elem in sortedElements],
-                                 headers=attributes,
-                                 tablefmt='rst'),
-                        caption='List of elements')
-
+.. only:: html
```

Comment:
You know we have a ticket open for this, right?

https://github.com/terrapower/armi/issues/1624

Reply:
@albeanth @mgjarrett This is certainly one of the main options in the ticket I linked.  But I thought maybe we would discuss it first.  I certainly think it doesn't belong in the API. But it could be somewhere else in the docs.

Thoughts?

Reply:
I think this is a fine band-aid solution but shouldn't be the long term solution. 

Having a 109-page long table in the HTML is also not very useful. I think we should either 1) get rid of it, or 2) have a small table with ~10 nuclides as an example and reference the rest in a script that can be run.